mayastor/src/bdev/nexus/nexus_rpc.rs

```diff
@@ -13,6 +13,7 @@ use rpc::mayastor::{
     PublishNexusRequest,
     RemoveChildNexusRequest,
     UnpublishNexusRequest,
+    ShareProtocolNexus,
```

Comment:
I've commented on the proto file -- but to be sure I'll add it here as well. The enum variants are the same for `ShareProtocol` as well as `ShareProtocolNexus` Why cant we use the same enum?